src/utils/utils.ts

```diff
@@ -244,9 +242,13 @@ export function ensureFuseBoxPath(input: string) {
 
 export function joinFuseBoxPath(...any): string {
   let includesProtocol = any[0].includes('://');
+  let isRelative = any[0].includes('./');
   let joinedPath = !includesProtocol
     ? path.join(...any)
     : any[0].replace(/([^/])$/, '$1/') + path.join(...any.slice(1));
+  if (isRelative) {
+    joinedPath = './' + joinedPath;
```

Comment:
Oh oh... You can't do that. That will break a lot of things, starting from the client side
`__file("hello/foo.bar")` will become `__file("./hello/foo.bar")` and everything will get broken

Reply:
So, what's your idea on solving this? :) we could fix this only for the use of webIndex etc. on the return side of the call